packages/grafana-runtime/src/services/backendSrv.ts

```diff
@@ -126,7 +126,7 @@ export interface FetchError<T = any> {
   traceId?: string;
 }
 
-export function isFetchError(e: unknown): e is FetchError {
+export function isFetchError<T = any>(e: unknown): e is FetchError<T> {
```

Comment:
Added this to betterer - Ideally this would be `T = unknown`, but don't want to break anyone currently unsafely relying on this being `any` in this PR.